cmd/kured/main.go

```diff
@@ -245,6 +253,12 @@ func drain(client *kubernetes.Clientset, node *v1.Node) {
 		}
 	}
 
+	if notifyURL != "" {
```

Comment:
For me there should be no duplication of slackHookURL.

We need something to parse the CLI arguments earlier in the code, and a single way to send notifications instead.
Retro compatibility should come from parsing the arguments and converting those to the new mechanism instead.

Reply:
Also, if you can _remove_ code that would be deprecated by that change, that would be awesome

Reply:
Removed.